code/lib/builder-vite/src/transform-iframe-html.ts

```diff
@@ -22,7 +21,6 @@ export async function transformIframeHtml(html: string, options: ExtendedOptions
   }));
 
   return html
-    .replace('<!-- [TITLE HERE] -->', title || 'Storybook')
```

Comment:
Yes, this is what my note in the description is about:

>Replaced our custom ExtendedOptions with the standard Options type, and removed customization of the iframe title. I am not convinced this was working, and I'm not sure why it would be needed. We can probably find a good way to accomplish this in a builder-agnostic way in the future if it is requested.

It looks like the code was expecting a `title` to be provided in `options`, but it's not in the TypeScript type for `Options`, so I'm not sure it ever worked.  Unless the types are wrong?